Assets/_BForBoss/_Character/Scripts/PlayerMovementBehaviour.cs

```diff
@@ -51,6 +66,15 @@ public override float GetMaxAcceleration()
             return IsWallRunning ? _wallRunBehaviour.GetMaxAcceleration() : base.GetMaxAcceleration();
         }
         
+        public void SetControlConfiguration()
```

Comment:
This is to Set the Control Configuration. 

In the `PlayerBehaviour` script, whenever it's in playmode, it sets the control configuration here. 
https://github.com/PerigonGames/BForBoss/pull/249/files#diff-d762f663150a59c435e08aed3b01818e9a125c4b1152df922fa51bf3206fe13eR59-R65